modules/@apostrophecms/ui/ui/apos/components/AposButton.vue

```diff
@@ -501,7 +501,7 @@ export default {
       transform: none;
       box-shadow: none;
       outline: none;
-      border: 0;
+      border-color: transparent;
```

Comment:
I probably messed this up to start but `no-motion` should not unset a border .. there is a `no-border` modifier we should use. Remove this behavior